Views/Shared/_Layout.cshtml

```diff
@@ -2,6 +2,9 @@
 @using Personaltool.Models
 @inject SignInManager<ApplicationUser> SignInManager
 @inject UserManager<ApplicationUser> UserManager
+@{
+    ViewData["Privacy"] = "Datenschutzerklärung";
```

Comment:
Why is this is the _Layout? I don't see a reason for it, if we want localization we should use a proper solution